setup.py

```diff
@@ -44,32 +41,39 @@ def package_data(pkg, root_list):
 
 BLOCKS = [
     'mentoring = mentoring:MentoringBlock',
-    'mentoring-dataexport = mentoring:MentoringDataExportBlock',
-]
 
-BLOCKS_CHILDREN = [
     'mentoring-table = mentoring:MentoringTableBlock',
-    'column = mentoring:MentoringTableColumnBlock',
-    'header = mentoring:MentoringTableColumnHeaderBlock',
+    'mentoring-column = mentoring:MentoringTableColumn',
     'answer = mentoring:AnswerBlock',
-    'quizz = mentoring:QuizzBlock',
-    'message = mentoring:MentoringMessageBlock',
-    'tip = mentoring:QuizzTipBlock',
-    'choice = mentoring:QuizzChoiceBlock',
-    'html = mentoring:HTMLBlock',
+    'answer-recap = mentoring:AnswerRecapBlock',
+    'mcq = mentoring:MCQBlock',
+    'rating = mentoring:RatingBlock',
+    'mrq = mentoring:MRQBlock',
+    'mentoring-message = mentoring:MentoringMessageBlock',
+    'tip = mentoring:TipBlock',
```

Comment:
@bradenmacdonald I'm not sure I'd want to rename the repo right now, since we are already pretty advanced in the review process, that might complicate things by spreading reviews over different locations. I would likely do this in a later step - maybe at the same time as we would move the children to their own repo too? For now I'd be happy with just prefixing the children tags with pb- to not have to migrate things later on.